packages/styles/src/_core.scss

```diff
@@ -3,3 +3,4 @@
 @forward './mixins';
 
 @import './themes/bootstrap/core';
+@import './elements';
```

Comment:
I'd say, this is definitively the wrong place to integrate the elements files.
And please do not use `@import` statements (it's deprecated and we only use it to import bootstrap files, because they can not be used in a different way). Instead, use `@use` or `@forward`.

In my opinion, we need to import it here:
- `styles/src/base.scss`
- `styles/src/index.scss`
- `styles/src/intranet.scss`
- `styles/src/post-external.scss`
- `styles/src/post-internal.scss`

Reply:
The `elements` style is moved to mentioned places.
The `elements` style is moved to `styles/src/components/base` for now and it should be moved/removed when the `/root` and `.reboot` are cleaned-up.

Reply:
The `elements` is moved to `styles/src/basic` instead of `base`.